store_app/business/user_edit_funcs.py

```diff
@@ -17,3 +17,9 @@ def change_username(user_id, new_username):
     user = User.query.get(user_id)
     user.username = new_username
     db.session.commit()
+
+
+def change_password(user_id, new_password):
```

Comment:
You could consider to do like most applications do, ask for the old password and update the password only if the old one is correct. Not necessary but a nice to have 😄

Reply:
I have added the condition to check for old password now.